tests/test_hdf5_backend.py

```
@@ -160,6 +160,53 @@ def test_defaultCompressionIsGzip(self, tmp_path: Path) -> None:
+    def test_hdf5pluginAutoRegistered(self) -> None:
+        """Importing the HDF5 backend should register hdf5plugin filters.
+
+        Uses a subprocess for a clean interpreter — otherwise pytest's own
+        imports might pollute sys.modules and hide the bug this fix prevents.
+        """
+        pytest.importorskip("hdf5plugin")
+        import subprocess
+        import sys
+
+        code = "from versionable import _hdf5_backend; import sys; assert 'hdf5plugin' in sys.modules"
+        result = subprocess.run([sys.executable, "-c", code], capture_output=True, text=True, check=False)
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)